src/openms/include/OpenMS/CHEMISTRY/EmpiricalFormula.h

```diff
@@ -126,6 +127,21 @@ namespace OpenMS
     double getAverageWeight() const;
 
     /**
+      @brief Fills this EmpiricalFormula with an approximate elemental composition for a given average weight and approximate elemental stoichiometry
+
+      @return integer return flag describing result of the approximation. 0 = no problems, 1 = negative hydrogens requested.
```

Comment:
can you move that below the `@params`?

The following is a suggestion, not a request:
Using an Int is a matter of taste I guess, since the return values are never clear intuitively (the Windows API drives me nuts every time).
But here, I'd have to write
`if (ef.estimateFromWeightAndComp(...) == 1) { // deal with hydrogens ...}`

whereas for a bool (true = success, false = fail) I'd have
`if (!ef.estimateFromWeightAndComp(...)) { // deal with hydrogens ...}`

I find the latter slightly more intuitive, but its a matter of taste ofc.